test/integration/tcp_tunneling_integration_test.cc

```diff
@@ -498,6 +518,64 @@ TEST_P(ProxyingConnectIntegrationTest, ProxyConnect) {
   cleanupUpstreamAndDownstream();
 }
 
+TEST_P(ProxyingConnectIntegrationTest, ProxyExtendedConnect) {
+#ifdef ENVOY_ENABLE_UHV
+  // TODO(#24945): This test needs CONNECT/upgrade normalization code which is not yet
+  // available in UHV.
+  return;
+#endif
+  add_upgrade_config_ = true;
+  initialize();
+
+  // Send request headers.
+  codec_client_ = makeHttpConnection(lookupPort("http"));
+  auto encoder_decoder = codec_client_->startRequest(extended_connect_headers_h2_);
```

Comment:
If this is only used inone test, it can be a local variable?

Reply:
Done